### PR TITLE
Added ekta chain and ekta chain testnet to imports

### PR DESCRIPTION
**Pull request by begmuhommet**
## Description

_Concise description of proposed changes_

## Additional Information

- [ ] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

File: pnpm-lock.yaml
[the remark this answers is not in the document]
Comment (begmuhommet) [reply pnpm-lock.yaml:1]: Changeset removed

[the remark this answers is not in the document]
Comment (tmm) [reply pnpm-lock.yaml:1]: We want to keep `.changeset`. I meant to remove your changes to the `pnpm-lock.yaml`. Once you restore `.changeset`, will be good to go.

[the remark this answers is not in the document]
Comment (begmuhommet) [reply pnpm-lock.yaml:1]: Done. Thanks

File: packages/chains/src/index.ts [hunk 1]
```diff
@@ -90,3 +90,5 @@ export { xdcTestnet } from './xdcTestnet'
 export { zhejiang } from './zhejiang'
 export { zkSync } from './zkSync'
 export { zkSyncTestnet } from './zkSyncTestnet'
+export { ekta } from './ekta'
+export { ektaTestnet } from './ektaTestnet'
```
Comment (tmm) [inline packages/chains/src/index.ts:94]: Missed this earlier. Let's keep this in alphabetical order.

**Review by tmm — CHANGES_REQUESTED** [review]

File: packages/chains/src/index.ts (hunk 1 above)
Comment (begmuhommet) [reply packages/chains/src/index.ts]: Sure. Done

**Review by tmm — APPROVED** [review]

**Pull request merged**